create_infotexts.py

```diff
@@ -229,6 +231,32 @@ def __init__(self):
 
         self.meta_cats.extend(batch_cats)
 
+    def add_cat_archaeological_exhibition_based_on_description(self, description):
+        """
+        Special case fix to add content category in special cases from <Beskrivning>.
+        
+        See task: https://phabricator.wikimedia.org/T163317
+        
+        Populates self.content_cats
+        
+        :return: None
+        """
+        if "utställning" in description.lower():
+            self.content_cats.append("Archaeological_exhibitions")
+
+    def add_cat_interior_of_tombs_based_on_description(self, description):
+        """
+        Special case fix to add content category in special cases from <Beskrivning>.
+
+        See task: https://phabricator.wikimedia.org/T163316
+
+        Populates self.content_cats
+
+        :return: None
+        """
+        if "interiör" in description and "grav" in description.lower():
```

Comment:
Do they? With every word in description .lower() there can't be a match for "Interiör"?